### PR TITLE
New layout & graph

### PR DESCRIPTION
**Pull request by katelkq**

### CODE REVIEW

File: frontend/src/types/status.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-shadow */
+export enum Status {
```
Comment (mileswatson) [inline frontend/src/types/status.ts:2]: Generally better to use literal types than adding an enum, to make comparing Python types and JS types easier. Also, I don't believe there is a need for a "pending" order type, as they should go through (or not) in the same request.

File: frontend/src/routes/explore/+page.svelte [hunk 2]
```diff
@@ -119,20 +126,36 @@
...
             </div>
 
-            <div class="level-right">
-                <div class="level-item">
-                    <span class="icon">
-                        <a href="/"><i class="fas fa-magnifying-glass" /></a>
-                    </span>
+            <div class="level-right" style="width: 60%">
+                <div class="level-item mr-3" style="width: 100%">
+                    <button class="button is-rounded" style="width: 100%">
```
Comment (mileswatson) [inline frontend/src/routes/explore/+page.svelte:131]: Is this button intended to be clickable?

File: frontend/src/routes/explore/+page.svelte [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <script lang="ts">
     import Watchlist from "$components/Watchlist.svelte";
     import News from "$components/News.svelte";
+    import Graph from "$components/Graph.svelte";
+
+    let w = 0;
```
Comment (mileswatson) [inline frontend/src/routes/explore/+page.svelte:6]: `w` and `h` aren't very descriptive, would be better to have `graphWidth` and `graphHeight`.

File: frontend/src/types/status.ts (hunk 1 above)
Comment (katelkq) [reply frontend/src/types/status.ts:2]: I think you've fixed the typing here a while ago and this file is just as-yet-undeleted zombie file :D
"Pending" is there as I'm not sure how the api works - we can delete that once we've got it attached?

File: frontend/src/routes/explore/+page.svelte (hunk 2 above)
Comment (katelkq) [reply frontend/src/routes/explore/+page.svelte:131]: Yep the entire search bar is actually just one button

File: frontend/src/types/status.ts (hunk 1 above)
Comment (mileswatson) [reply frontend/src/types/status.ts:2]: Got it :)

**Comment by katelkq** [comment]
I'll close this branch down and re-open a PR with an updated branch

**Pull request closed without merging**